src/common/base64.ts

```diff
@@ -0,0 +1,17 @@
+export function urlDecode(a: string): string {
+  return atob(a).replace(/_/g, "/").replace(/-/g, "+")
```

Comment:
this line was
```ts
  return atob(a).replace(/\_/g, "/").replace(/\-/g, "+")
```
but eslint told me those escape characters weren't necessary. still seems to work fine